packages/autorest.python/autorest/codegen/models/code_model.py

```diff
@@ -290,3 +290,59 @@ def need_typing_extensions(self) -> bool:
         ):
             return True
         return False
+
+    @property
+    def import_core(self) -> str:
```

Comment:
I still think all of this import logic should be on the `ImportSerializer`, I don't think having import stuff on the code model looks good.

Maybe instead of
```python
file_import.add_submodule_import(
    "azure.core.rest",
    "AsyncHttpResponse",
    ImportType.AZURECORE,
    TypingSection.CONDITIONAL,
)
```
we can do
```python
 file_import.add_submodule_import(
    "rest",
    "AsyncHttpResponse",
    ImportType.SDKCORE,
    TypingSection.CONDITIONAL,
)
```

And in the import serializer, since we know this is a "core" import, we can either add an import for `corehttp.rest` if unbranded else `azure.core.rest` if not. And this will apply to all "core" imports